narrative_llm_agent/agents/analyst.py

```diff
@@ -6,62 +6,138 @@
 from langchain.vectorstores import Chroma
 from langchain.memory import ConversationBufferMemory, ReadOnlySharedMemory
 from langchain.chains import RetrievalQA
-from langchain.tools import BaseTool, tool
+from langchain.tools import tool
+import os
+from pathlib import Path
 
 
 class AnalystInput(BaseModel):
-    input: str = Field(description="query to look up KBase documentation, catalog or tutorials")
+    input: str = Field(
+        description="query to look up KBase documentation, catalog or tutorials"
+    )
+
+DEFAULT_CATALOG_DB_DIR: Path = Path(__file__).parent / "vector_db_app_catalog"
```

Comment:
Set the default paths here.
This PR also migrates the folders from `tests/agents` to `agents`